api/handler.go

```diff
@@ -446,9 +446,11 @@ func (h *apiHandler) resolveRealtimeQuerySpec(r *http.Request) (views.QuerySpec,
 		return views.QuerySpec{}, http.StatusBadRequest, []error{errors.New("param 'to' cannot be specified if 'from' is not defined")}
 	}
 	if spec.From != nil || spec.To != nil {
-		if spec.To == nil {
-			now := time.Now()
-			spec.To = &now
+		// If using time range, then we allow to query max 1 min before now(),
+		// because the current "per minute" aggregation may not be finalized yet
+		lastToAllowed := time.Now().Add(-1 * time.Minute)
```

Comment:
Let me test my understanding: 
if `to` parameter is unspecific, you set it to "now" minus 1 minute and then pass that into the query? 

My preferred method would be to add this to the query `and timestamp_ts < toStartOfMinute(now())` but i'll also trust you that these are doing effectively the same thing. 

I also added this to the notion doc https://www.notion.so/livepeer/Viewership-Queries-e27bdb9df64240cea371ca4e0f11c0f6?pvs=4#4f610f7bd8a3413d8cc2ff6a0f52b966

I

Reply:
Yes, IMO it's the same.